zwave_js/config.json

```diff
@@ -21,7 +21,7 @@
     "network_key": ""
   },
   "schema": {
-    "device": "device(subsystem=tty)",
+    "device": "str",
```

Comment:
This change makes sense in a way, I get the reasoning. However, we do want to think to be more fool proof. Considering this platform; using the `device()` validator makes sense here.

This also is closely related to the Z-Wave JS Core integration, that sets this up. So that is something to keep in mind here, as all changes to this schema, effects the core integration as well.

As an alternative, a separate option could be made, however, that has the same challenges ( Core configures), and additionally adds that both options become "optional" in order to be able to select one or the other.

At this point, I think this is a choice between evils. And I think we need to get a decision from the core people involved with Z-Wave JS, the Supervisor & add-ons in order to get this going. Maybe this should be an architectural discussion instead?